tests/runner.py

```diff
@@ -1502,7 +1502,12 @@ def reftest(self, expected, manually_trigger=False):
       }
 ''' % (reporting.read(), basename, int(manually_trigger)))
 
+  def skip_if_wasm_backend_pthreads_js(self, args):
+    if self.is_wasm_backend() and 'USE_PTHREADS=1' in args and 'WASM=0' in args:
```

Comment:
Also `-pthread`, probably.

Reply:
Do you mean we should check both? (using and or or?) Why?

Reply:
I mean checking for either. I forget if this is implemented yet, but `-pthread` should imply `-s USE_PTHREADS=1` and vice versa, so tests should be free to use either one and not the other. I guess this isn't relevant if `args` here is the processed `newargs`, but I don't think that's the case.

Reply:
This code sees the arguments the test passes to emcc. Currently they all pass the old form, but yeah, sounds good to support the new one too, so we can update the tests eventually if we want.